tests/Oryx.RuntimeImage.Tests/DotnetCoreImageVersionsTest.cs

```diff
@@ -75,8 +75,8 @@ public void DotnetVersionMatchesImageName_NetCoreApp1Versions(string version, st
 
         [Theory]
         [InlineData("2.0", "Version  : 2.0.9")]
-        [InlineData("2.1", "Version: 2.1.8")]
-        [InlineData("2.2", "Version: 2.2.2")]
+        [InlineData("2.1", "Version: 2.1.9")]
+        [InlineData("2.2", "Version: 2.2.3")]
```

Comment:
in the YAML, it's 2.1.602 & 2.2.3 - why are they different?

Reply:
2.1.602 is for sdk <build image> 2.2.3 is for runtime image, I may need to change some tests in buildimage as I'm not able to test is locally, waiting to see which tests are failing in buildimage